tests/mctpenv/__init__.py

```diff
@@ -37,13 +38,26 @@ def to_nlmsg(self, seq = 0):
             resp['attrs'] = [['NLMSGERR_ATTR_MSG', self.msg]]
         return resp
 
+class PhysicalBinding(enum.Enum):
+    UNSPEC = 0x00
+    SMBUS = 0x01
+    PCIE_VDM = 0x02
+    USB = 0x03
+    KCS = 0x04
+    SERIAL = 0x05
+    I3C = 0x06
+    MMBI = 0x07
+    PCC = 0x08
+    UCIE = 0x09
+    VENDOR = 0xFF
+
 class System:
     class Interface:
         """Interface constructor.
 
         Initial mtu is set to max_mtu.
         """
-        def __init__(self, name, ifindex, net, lladdr, min_mtu, max_mtu, up = False):
+        def __init__(self, name, ifindex, net, lladdr, min_mtu, max_mtu, up = False, phys_binding = PhysicalBinding.UNSPEC):
```

Comment:
please wrap to 80 cols. I'll add some proper python formatting at some point, but best to keep with rough conventions in the meantime.